audio-clips/src/main/java/org/odk/collect/audioclips/AudioClipViewModel.kt

```diff
@@ -162,11 +159,13 @@ class AudioClipViewModel(private val mediaPlayerFactory: Supplier<MediaPlayer>,
     private fun schedulePositionUpdates() {
         positionUpdatesCancellable = scheduler.repeat(
             {
-                _mediaPlayer?.let {
-                    val currentlyPlaying = currentlyPlaying.value
-                    if (currentlyPlaying != null) {
-                        val position = getPositionForClip(currentlyPlaying.clip.clipID)
-                        position.postValue(it.currentPosition)
+                val currentlyPlaying = currentlyPlaying.value
+                if (currentlyPlaying != null) {
+                    val position = getPositionForClip(currentlyPlaying.clip.clipID)
```

Comment:
This could be moved down so that it is only set up if `currentPosition != null`.